src/services/widgets.data.service.ts

```diff
@@ -16,7 +16,7 @@ export const fetchWidgets = async (id: number, lang: string, yearAgo?: number):
     if (yearAgo) {
       query.push(`years_ago=${yearAgo}`);
     }
-    const widgetsUrl = `${NEWS_FLASH_API}?${query.join('&')}`;
+    const widgetsUrl = `${NEWS_FLASH_API}?${query.join('&')}&mock=${process.env.REACT_APP_SHOW_MOCK_INFORMATION}`;
```

Comment:
`&mock` should only appear in URL if process.env.REACT_APP_SHOW_MOCK_INFORMATION is `true`